day_two_guide/topics/scaling_etcd.adoc

```diff
@@ -0,0 +1,432 @@
+////
+Scaling etcd
+
+Module included in the following assemblies:
+
+* day_two_guide/host_level_tasks.adoc
+////
+
+Scaling the etcd cluster can be performed vertically by adding more resources to
+the etcd hosts, or horizontally by adding more etcd hosts.
+
+[NOTE]
+====
+Due to the voting system etcd uses, the cluster must always contain an odd
+number of members.
+====
+
+The new host requires a fresh RHEL7 dedicated host. The etcd storage should be
```

Comment:
Should these be prereqs?
I'd spell out RHEL
I'd replace the second sentence with something like "For best performance, use a dedicated SSD disk that is mounted in `/var/lib/etcd` for the etcd storage."